components/mpas-ocean/src/mode_init/mpas_ocn_init_overflow.F

```diff
@@ -228,8 +228,17 @@ subroutine ocn_init_setup_overflow(domain, iErr)!{{{
 
            ! Set layerThickness and restingThickness
            if ( trim(config_overflow_layer_type) == 'z-level' ) then
+              layerThickness(:,iCell) = 0.0_RKIND
```

Comment:
is this necessary?  When I plotted the layerThickness out of initial_state it was nan below maxLevelCell, but perhaps this gets filled appropriately in the forward step